hbase_sql/src/test/java/com/splicemachine/derby/utils/ProgressThreadIT.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2020 - 2021 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.splicemachine.derby.utils;
+
+import com.splicemachine.db.shared.ProgressInfo;
+import com.splicemachine.db.impl.tools.ij.ProgressThread;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+
+public class ProgressThreadIT {
```

Comment:
This should be a Test, not an IT (ITs depend on an instance of the DB running)

Reply:
done